doc/doc-site/docs/off-chain/rust-sdk.md

```diff
@@ -1,3 +1,255 @@
+```
+
+Next, we'll fund the accounts with some example coins (eAPT and eUSDC). To do so, we use the `fund` helper function (you can find its definition in the script's source).
+
+```rust
+fund(&e_apt, 10u64.pow(18), &mut econia_client_a, faucet_address).await?;
+
+fund(&e_usdc, 10u64.pow(10), &mut econia_client_a, faucet_address).await?;
+```
+
+Now that that's out of the way, we'll register the account to the market. This is an important step that all accounts have to do before participating in a market. For this, we'll use the `register_market_account` function that will create the appropriate `EntryFunction` for us. We then submit it.
+
+```rust
+let entry = register_market_account(econia_address, &e_apt, &e_usdc, market_id, 0)?;
+econia_client_a.submit_tx(entry).await?;
+```
+
+We'll deposit some coins to be used to open orders. For this, we use `deposit_from_coinstore`.
+
+```rust
+let e_apt_subunits = 10u64.pow(9);
+let entry = deposit_from_coinstore(econia_address, &e_apt, market_id, 0, e_apt_subunits)?;
+econia_client_a.submit_tx(entry).await?;
+
+let e_usdc_subunits = 10u64.pow(10);
+let entry = deposit_from_coinstore(econia_address, &e_usdc, market_id, 0, e_usdc_subunits)?;
+```
+
+Account A is now set up and ready to be used.
+
+### Step 3: Place two limit orders with account A
+
+As the title says, it's showtime. Let's place some orders.
+
+```rust
+let buy_base_lots = 10u64.pow(3);
+let buy_ticks_per_lot = 10u64.pow(3);
+let entry = place_limit_order_user_entry(econia_address, &e_apt, &e_usdc, market_id, &econia_address, Side::Bid, buy_base_lots, buy_ticks_per_lot, Restriction::NoRestriction, SelfMatchBehavior::CancelMaker)?;
+econia_client_a.submit_tx(entry).await?;
+
+let sell_base_lots = 10u64.pow(3);
+let sell_ticks_per_lot = 2 * 10u64.pow(3);
+let entry = place_limit_order_user_entry(econia_address, &e_apt, &e_usdc, market_id, &econia_address, Side::Ask, sell_base_lots, sell_ticks_per_lot, Restriction::NoRestriction, SelfMatchBehavior::CancelMaker)?;
+econia_client_a.submit_tx(entry).await?;
+```
+
+This is a bit more complex than previous code. Let's inspect the `place_limit_order_user_entry` function. It's first argument, as all first arguments of functions in the `entry` module is the Econia contract address. Then comes two type arguments, base and quote. These are Aptos type arguments, you can see in the `init` function how to create a type argument, or look into the Aptos docs for more details. Next we have the market ID which is a simple integer. Then comes the integrator. You can read more about integrators [here](), but for now, we'll just set this to the Econia address. We then have side, size and price which are all pretty self-explanatory. Next up is restrictions and self match behavior. You can read more about each of these enums variants [here](https://github.com/econia-labs/econia/blob/6256710b2ebe306d2861e6d02d72f95373500ab6/src/move/econia/sources/market.move#L859-L942).
```

Comment:
(So this comment actually refers to the final sentence of this paragraph, but the paragraph is all in one line, hence my above suggestion to line break each sentence in docs pages)

I think permalinks to code like these are fine as opposed to sample code snippets that are copy-pasted, but it might also be best in this case to refer to something like the `market.move` file without a link and then just say constants under the constants section

Reply:
For the integrators link I would do a link to the docusaurus file, something like `[../overview/incentives]`